docs/guide/robot_sbc/setup_jetson_nano.md

```diff
@@ -66,76 +136,364 @@ pip3 install -U gdown
+
+```bash
+sudo apt-get install python-dev libsdl1.2-dev libsdl-image1.2-dev libsdl-mixer1.2-dev libsdl-ttf2.0-dev libsdl1.2-dev libsmpeg-dev python-numpy subversion libportmidi-dev ffmpeg libswscale-dev libavformat-dev libavcodec-dev libfreetype6-dev
+pip install pygame
 ```
 
-Optionally, you can install the RPi.GPIO clone for Jetson Nano from [here](https://github.com/NVIDIA/jetson-gpio). This is not required for default setup, but can be useful if using LED or other GPIO driven devices.
+Later on you can add the `CAMERA_TYPE="WEBCAM"` in myconfig.py.
 
 
 
-##  Step 5: Install Donkeycar Python Code
+## Installation for Donkey Car main
 
-* Change to a dir you would like to use as the head of your projects. Assuming you've already made the `projects` directory above, you can use that:
+Instructions for the latest code from the `main` branch. Note the 
+installation differs between the two available OSs. On Jetson Nano you will 
+need to install Jetpack 4.6.2. If you have newer Jetson board or a Jetson 
+Xavier you need to install Jetpack 5.0.2.
+
+
+### Installation on Jetson Nano
+
+
+* [Step 1b: Flash Operating System](#step-1b-flash-operating-system)
+* [Step 2b: Free up the serial port](#step-2b-free-up-the-serial-port-optional-only-needed-if-youre-using-the-robohat-mm1)
+* [Step 3b: Setup Python Environment](#step-3b-setup-python-environment)
+* [Step 4b: (Optional) Install PyGame for USB camera](#step-4b-optional-install-pygame-for-usb-camera)
+
+
+#### Step 1b: Flash Operating System
+
+These instructions work for Jetpack 4.6.1. 
+
+* If you have a 4gb Jetson Nano then download Jetpack 4.6.1 from Nvidia
+  here: [jetson-nano-jp461-sd-card-image.zip](https://developer.nvidia.com/embedded/l4t/r32_release_v7.1/jp_4.6.1_b110_sd_card/jeston_nano/jetson-nano-jp461-sd-card-image.zip)
+* If you have a 2gb Jetson Nano the download Jetpack 4.5.1from Nvidia
+  here: [jetson-nano-2gb-jp461-sd-card-image.zip](https://developer.nvidia.com/embedded/l4t/r32_release_v7.1/jp_4.6.1_b110_sd_card/jetson_nano_2gb/jetson-nano-2gb-jp461-sd-card-image.zip)
+
+
+Visit the official [Nvidia Jetson Nano Getting Started Guide](https://developer.nvidia.com/embedded/learn/get-started-jetson-nano-devkit#prepare)
+or [Nvidia Xavier NX Getting Started Guide](https://developer.nvidia.com/embedded/learn/get-started-jetson-xavier-nx-devkit).
+Work through the __Prepare for Setup__, __Writing Image to the microSD Card__,
+and __Setup and First Boot__ instructions, then return here.
+
+Once you're done with the setup, ssh into your vehicle. Use the terminal for
+Ubuntu or Mac. [Putty](https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html)
+for windows.
+
+
+Remove Libre Office:
 
 ```bash
-cd ~/projects
+sudo apt-get remove --purge libreoffice*
+sudo apt-get clean
+sudo apt-get autoremove
+```
+
+And add a 6GB swap file:
+
+```bash
+git clone https://github.com/JetsonHacksNano/installSwapfile
+cd installSwapfile
+./installSwapfile.sh
+reboot 
+```
```

Comment:
My output for `free -m` is:
```
              total        used        free      shared  buff/cache   available
Mem:           3960         593         125          27        3240        3140
Swap:          8124           1        8122
```

I'm not sure why it looks like 8GB swap, [the script does say 6](https://github.com/JetsonHacksNano/installSwapfile/blob/838403c20561f0d6ca840b25f6b46a460a8f87fe/installSwapfile.sh#L10)

Reply:
Ok thanks for reporting, maybe the script has changed? @TCIII - would you know? Anyway I have updated the doc. Further down for the Xavier we explicitly give 8GB as the size as the script was doing something different afaik.